more_itertools/more.py

```diff
@@ -2274,11 +2276,10 @@ def __reduce__(self):
         return numeric_range, (self._start, self._stop, self._step)
 
     def __repr__(self):
+        cls_name = self.__class__.__name__
```

Comment:
I take it `ruff` did not make this change? I would like it if you could list which things are being done by the automatic formatter and which things you are changing to match your own preferences.

Reply:
It is not `ruff`, indeed, I just saw it, but I can revert it this PR is not the right place for it.

Reply:
Also some manual work on the RET family (these if-elif-else, especially alongside return and raise statements and so on).